info.md

```diff
@@ -1,7 +1,11 @@
 ### Bellingham Information
-* Provide Group Information
+* Currently meeting virtually
+* [Become a Member](https://www.owasp.org/index.php/Membership)
+* [Local Chapter Support](https://owasp.org/donate)
+* [Speaker Agreement](https://owasp.org/www-policy/legal/speaker-agreement)
+* [Code of Conduct](https://owasp.org/www-policy/operational/conferences-events.html)
 
 ### Social Links
-* [Meetup](#)
+* [Meetup - coming soon](#)
 
```

Comment:
We should include a link to the TAG Discord, assuming we'll be using that for our initial conversation space.